activities.json

```diff
@@ -166,3 +166,31 @@
 {"activity":"Write a note of appreciation to someone","accessibility":0.0,"type":"social","participants":1,"price":0.0,"link":"","key":"1770521","enabled":true}
 {"activity":"Draw something interesting","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"","key":"8033599","enabled":true}
+{"activity":"Learn and play a new card game","accessibility":0.0,"type":"recreational","participants":1,"price":0.0,"link":"https://www.pagat.com","key":"9660022","enabled":true}
+{"activity":"Write a list of things you are grateful for","accessibility":0.0,"type":"relaxing","participants":1,"price":0.0,"link":"","key":"2062010","enabled":true}
+{"activity":"Organize one of your collections","accessibility":0.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"6098037","enabled":true}
+{"activity":"Learn Morse code","accessibility":0.0,"type":"education","participants":1,"price":0.0,"link":"https://en.wikipedia.org/wiki/Morse_code","key":"3646173","enabled":true}
+{"activity":"Go for a walk","accessibility":0.1,"type":"relaxation","participants":1,"price":0.0,"link":"","key":"4286250","enabled":true}
+{"activity":"Mow your neighbour's lawn","accessibility":0.2,"type":"charity","participants":1,"price":0.0,"link":"","key":"1303906","enabled":true}
+{"activity":"Compliment someone","accessibility":0.0,"type":"social","participants":2,"price":0.0,"link":"","key":"9149470","enabled":true}
+{"activity":"Draft your living will","accessibility":0.5,"type":"busywork","participants":1,"price":0.0,"link":"https://www.investopedia.com/terms/l/livingwill.asp","key":"2360432","enabled":true}
+{"activity":"Prepare a 72-hour kit","accessibility":0.5,"type":"busywork","participants":1,"price":0.5,"link":"https://www.ready.gov/kit","key":"4266522","enabled":true}
+{"activity":"Color","accessibility":0.0,"type":"relaxation","participants":1,"price":0.1,"link":"","key":"5322987","enabled":true}
+{"activity":"Organize your pantry","accessibility":0.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"3954882","enabled":true}
+{"activity":"Back up important computer files","accessibility":0.2,"type":"busywork","participants":1,"price":0.2,"link":"","key":"9081214","enabled":true}
+{"activity":"Memorize a favourite quote or poem","accessibility":0.8,"type":"education","participants":1,"price":0.0,"link":"","key":"9008639","enabled":true}
+{"activity":"Do something you used to do as a kid","accessibility":0.8,"type":"relaxation","participants":1,"price":0.1,"link":"","key":"8827573","enabled":true}
+{"activity":"Organize your basement","accessibility":0.9,"type":"busywork","participants":1,"price":0.0,"link":"","key":"8203595","enabled":true}
+{"activity":"Write a todo list for the coming weekend","accessibility":1.0,"type":"busywork","participants":1,"price":0.0,"link":"","key":"9487940","enabled":true}
```

Comment:
There's already "Make a to-do list for your week". This seems too similar and I'm not sure if we can change it enough to make it justifiably different.

Reply:
I somehow missed that, Sorry. Will remove that from the commit.